src/axom/inlet/inlet_utils.cpp

```diff
@@ -58,8 +64,9 @@ bool checkIfRequired(const axom::sidre::Group& target, axom::sidre::Group& root)
   {
     const std::string msg = fmt::format(
       "[Inlet] Invalid integer value stored in "
-      " boolean value named {0}",
-      target.getName());
+      " boolean value named {0} for flag '{1}'",
+      target.getName(),
+      flag);
     SLIC_WARNING(msg);
     setWarningFlag(&root);
     return false;

```

Comment:
I wonder if this should warn about an incorrect value,
but then return `static_cast<bool>(inValue)` instead of `false`?